product-service/handlers/catalog-batch-process.ts

```diff
@@ -0,0 +1,56 @@
+import { SQSEvent } from 'aws-lambda';
+import * as AWS from 'aws-sdk';
+import 'source-map-support/register';
+import generateDbConfig from '../db/db-config';
+import { Client } from 'pg';
+import { Product, ProductSchema } from '../db/models/Product';
+import { insertProductQuery, insertStockQuery } from '../db/queries'
+import { logger } from './utils';
+
+export const catalogBatchProcess = async (event : SQSEvent) => {
+  logger('catalogBatchProcess', event);
+  const dbClient = new Client(generateDbConfig(process));
```

Comment:
Just an advice: it would be better, if you separate your DB-interaction logic in a separate module, so you can reuse it both here and in POST-lambda